api.md

```diff
@@ -1036,6 +1037,63 @@ CALL distributed_exec('CREATE DATABASE dist_database', transactional => FALSE);
 
 ---
 
+## create_distributed_restore_point() [](create_distributed_restore_point)
+
+Create a named point for performing restore across a multi-node TimescaleDB cluster.
+
+This function is an analogue of [`pg_create_restore_point`](https://www.postgresql.org/docs/current/functions-admin.html) designed
```

Comment:
I suggest using the link to the table where the function is specified instead of the top-level page: 

```suggestion
This function is an analogue of [`pg_create_restore_point`](https://www.postgresql.org/docs/current/functions-admin.html#FUNCTIONS-ADMIN-BACKUP-TABLE) designed
```